### PR TITLE
Charter Security WG

### PR DESCRIPTION
**Pull request by vdeturckheim**
This PR adds the Security WG as a chartered WG.
This probably can't be merged until https://github.com/nodejs/security-wg/pull/295 is merged.

After this, is there anything else I should to to have this validated?

cc @mhdawson

### CODE REVIEW

**Comment by vsemozhetbyt** [comment]
It seems this TOC also needs updating:
https://github.com/nodejs/TSC/blob/master/WORKING_GROUPS.md#current-working-groups

File: WORKING_GROUPS.md [hunk 1]
```diff
@@ -434,6 +434,37 @@ Responsibilities include:
...
 
+### [Security](https://github.com/nodejs/security-wg)
+
+The Security Working Group manages all aspects and process linked to security for Node.js.
+
+Responsibilities include:
+* Define and maintain security policies and procedures for:
+  * the core Node.js project
+  * other projects maintained by the Node.js Foundation technical group
+* Work with the node security project to bring community vulnerability data into
```
[the remark this answers is not in the document]
Comment (Trott) [reply WORKING_GROUPS.md]: Perhaps unfortunately (but perhaps not), the Node Security Platform styles it `Node` and not `Node.js`.  See https://medium.com/npm-inc/npm-acquires-lift-security-258e257ef639.

Although I think we still need a change: `node security project` -> `Node Security Platform` (assuming I'm right about this referring to Node Security Platform).

File: WORKING_GROUPS.md [hunk 2]
```diff
@@ -434,6 +434,37 @@ Responsibilities include:
...
+  modules.
+* Work to set a high standard for the Node.js project. Possibly efforts could
+  include penetration testing, security reviews etc, review guidelines, coding
+  standards etc.
+* Review and recommend processes for handling of security reports (but not the
+  actual handling of security reports, which are reviewed by a group of people
+  directly delegated to by the TSC).
+* Define and maintain policies and procedures for the coordination of security
+  concerns within the external Node.js open source ecosystem.
+* Offer help to npm package maintainers to fix high-impact security bugs
```
[the remark this answers is not in the document]
Comment (Trott) [reply WORKING_GROUPS.md]: Missing period on line 457 too.

File: WORKING_GROUPS.md [hunk 3]
```diff
@@ -434,6 +434,37 @@ Responsibilities include:
...
+
+Responsibilities include:
+* Define and maintain security policies and procedures for:
+  * the core Node.js project
+  * other projects maintained by the Node.js Foundation technical group
+* Work with the node security project to bring community vulnerability data into
+  the foundation as a shared asset.
+* Set up processes and procedures and follow these to ensure the vulnerability
+  data is updated in an efficient and timely manner. For example, ensuring there
+  are well documented processes for reporting vulnerabilities in community
```
Comment (WaleedAshraf) [inline WORKING_GROUPS.md]: `well documented` -> `well-documented`

File: WORKING_GROUPS.md [hunk 4]
```diff
@@ -434,6 +434,37 @@ Responsibilities include:
...
+  modules.
+* Work to set a high standard for the Node.js project. Possibly efforts could
+  include penetration testing, security reviews etc, review guidelines, coding
+  standards etc.
+* Review and recommend processes for handling of security reports (but not the
+  actual handling of security reports, which are reviewed by a group of people
```
Comment (WaleedAshraf) [inline WORKING_GROUPS.md]: nit: Maybe it's more clear to change `handling ` -> `administration` ? Maybe not. 👍

File: WORKING_GROUPS.md [hunk 5]
```diff
@@ -434,6 +434,37 @@ Responsibilities include:
...
+  * other projects maintained by the Node.js Foundation technical group
+* Work with the node security project to bring community vulnerability data into
+  the foundation as a shared asset.
+* Set up processes and procedures and follow these to ensure the vulnerability
+  data is updated in an efficient and timely manner. For example, ensuring there
+  are well documented processes for reporting vulnerabilities in community
+  modules.
+* Work to set a high standard for the Node.js project. Possibly efforts could
+  include penetration testing, security reviews etc, review guidelines, coding
+  standards etc.
```
Comment (WaleedAshraf) [inline WORKING_GROUPS.md]: Missing Comma `standards etc.` -> `standards, etc.`

File: WORKING_GROUPS.md [hunk 6]
```diff
@@ -434,6 +434,37 @@ Responsibilities include:
...
+  * the core Node.js project
+  * other projects maintained by the Node.js Foundation technical group
+* Work with the node security project to bring community vulnerability data into
+  the foundation as a shared asset.
+* Set up processes and procedures and follow these to ensure the vulnerability
+  data is updated in an efficient and timely manner. For example, ensuring there
+  are well documented processes for reporting vulnerabilities in community
+  modules.
+* Work to set a high standard for the Node.js project. Possibly efforts could
+  include penetration testing, security reviews etc, review guidelines, coding
```
Comment (WaleedAshraf) [inline WORKING_GROUPS.md]: Remove multiple `etc`.
`security reviews etc` -> `security reviews`

File: WORKING_GROUPS.md [hunk 7]
```diff
@@ -434,6 +434,37 @@ Responsibilities include:
...
+  actual handling of security reports, which are reviewed by a group of people
+  directly delegated to by the TSC).
+* Define and maintain policies and procedures for the coordination of security
+  concerns within the external Node.js open source ecosystem.
+* Offer help to npm package maintainers to fix high-impact security bugs
+* Maintain and make available data on disclosed security vulnerabilities in:
+  * the core Node.js project
+  * other projects maintained by the Node.js Foundation technical group
+  * the external Node.js open source ecosystem
+* Promote improvement of security practices within the Node.js ecosystem
```
Comment (WaleedAshraf) [inline WORKING_GROUPS.md]: `Promote improvement` -> `Promote the improvement`

**Comment by vdeturckheim** [comment]
Thanks for the reviews. I updated the doc.

**Comment by mhdawson** [comment]
@nodejs/tsc it would be good to get more approvals.  Unless we get objections I'd plan to land 1 week from today.

File: WORKING_GROUPS.md [hunk 8]
```diff
@@ -434,6 +435,37 @@ Responsibilities include:
   backporting changes to these branches.
 * Define the policy for what gets backported to release streams.
 
+### [Security](https://github.com/nodejs/security-wg)
+
+The Security Working Group manages all aspects and process linked to security for Node.js.
```
Comment (cjihrig) [inline WORKING_GROUPS.md]: process -> processes

... and maybe...

security for Node.js -> Node.js security

File: WORKING_GROUPS.md [hunk 9]
```diff
@@ -434,6 +435,37 @@ Responsibilities include:
...
+
+Responsibilities include:
+* Define and maintain security policies and procedures for:
+  * the core Node.js project
+  * other projects maintained by the Node.js Foundation technical group
+* Work with the Node Security Platform to bring community vulnerability data into
+  the foundation as a shared asset.
+* Set up processes and procedures and follow these to ensure the vulnerability
```
Comment (cjihrig) [inline WORKING_GROUPS.md]: I think most of this first sentence could be dropped and start with "Ensure that the..."

File: WORKING_GROUPS.md [hunk 10]
```diff
@@ -434,6 +435,37 @@ Responsibilities include:
...
+  data is updated in an efficient and timely manner. For example, ensuring there
+  are well-documented processes for reporting vulnerabilities in community
+  modules.
+* Work to set a high standard for the Node.js project. Possibly efforts could
```
Comment (cjihrig) [inline WORKING_GROUPS.md]: I'd drop this bullet point.

File: WORKING_GROUPS.md [hunk 11]
```diff
@@ -434,6 +435,33 @@ Responsibilities include:
...
 * Define the policy for what gets backported to release streams.
 
+### [Security](https://github.com/nodejs/security-wg)
+
+The Security Working Group manages all aspects and processes linked to Node.js security.
+
+Responsibilities include:
+* Define and maintain security policies and procedures for:
+  * the core Node.js project
+  * other projects maintained by the Node.js Foundation technical group
```
Comment (thefourtheye) [inline WORKING_GROUPS.md]: "Node.js Foundation technical group" - Does this mean TSC?

File: WORKING_GROUPS.md [hunk 12]
```diff
@@ -434,6 +435,33 @@ Responsibilities include:
...
+  concerns within the external Node.js open source ecosystem.
+* Offer help to npm package maintainers to fix high-impact security bugs.
+* Maintain and make available data on disclosed security vulnerabilities in:
+  * the core Node.js project
+  * other projects maintained by the Node.js Foundation technical group
+  * the external Node.js open source ecosystem
+* Promote the improvement of security practices within the Node.js ecosystem.
+* Recommend security improvements for the core Node.js project.
+* Facilitate and promote the expansion of a healthy security service and product
+  provider ecosystem vulnerabilities.
```
Comment (thefourtheye) [inline WORKING_GROUPS.md]: I couldn't wrap my head around this point. It makes sense without the 'vulnerabilities' at the end though.

**Comment by mhdawson** [comment]
Going to land as I believe I updated to address the remaining comments.  @thefourtheye if I've not addressed your comments adequately just let me know and I'll open a PR to further refine.

**Comment by mhdawson** [comment]
landed as b82207b0effbca80d47993d3896b7febb16104c1

**Comment by Trott** [comment]
Website needs to be updated too at https://nodejs.org/en/about/working-groups/ if there's not already a PR for that. @nodejs/website 

By the way, while adding stuff to that page, it might not be a terrible idea to take the time to alphabetize the list of working groups. It seems to be unordered.

**Comment by mhdawson** [comment]
PR to add minutes to website https://github.com/nodejs/nodejs.org/pull/1708 including alphabetization.

**Pull request closed without merging**